src/errors.js

```diff
@@ -536,4 +536,15 @@ Sk.abstr.setUpInheritance("StopIteration", Sk.builtin.StopIteration, Sk.builtin.
 goog.exportSymbol("Sk.builtin.StopIteration", Sk.builtin.StopIteration);
 
 
+// TODO: Extract into sys.exc_info(). Work out how the heck
+// to find out what exceptions are being processed by parent stack frames...
```

Comment:
Indeed not. My guess is a stack of global exception state, populated by the `catch` handle, which `sys.exc_info()` can then read. One for another PR.